content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -129,17 +129,23 @@ Plugins wishing to perform "pre-reserve" work should use the
 NormalizeScore extension point.
 {{< /note >}}
 
-### Reserve
-
-This is an informational extension point. Plugins which maintain runtime state
-(aka "stateful plugins") should use this extension point to be notified by the
-scheduler when resources on a node are being reserved for a given Pod. This
-happens before the scheduler actually binds the Pod to the Node, and it exists
-to prevent race conditions while the scheduler waits for the bind to succeed.
-
-This is the last step in a scheduling cycle. Once a Pod is in the reserved
-state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
+### Reserve {#reserve}
+
+The reserve extension point happens before the scheduler actually binds a Pod to
+a node and its uses are three-fold. First, it serves as a hook to allocate
+resources in the designated node that the scheduled Pod might need.  Second, it
+serves as an informational plugin which maintain runtime state (aka "stateful
+plugins") to be notified by the scheduler when resources on a node are being
+reserved for a given Pod. Third, it exists to prevent race conditions while the
+scheduler waits for the bind to succeed. This is the penultimate step in the
```

Comment:
Why do you say penultimate?

Reply:
There's also a permit extension point in the scheduling cycle -- reserve is not the last step.

Reply:
Or is permit a part of the binding cycle? The flow chart in this doc says it's a part of the scheduling cycle, but your other comment says it's a part of the binding cycle.

Reply:
Oh, I forgot about #88199. Please make sure the doc for Permit is up to date. I believe we did update it.